client/src/includes/tabs.test.js

```diff
@@ -0,0 +1,96 @@
+import {
+  TabInterface,
+  initDefaultTabs,
+} from './tabs';
+
+const BASIC_TAB_HTML = `
+  <div id="tabs" role="tablist">
+    <a href="#t1" role="tab" id="t1-label">Tab 1</a>
+    <a href="#t2" role="tab" id="t2-label">Tab 2</a>
+  </div>
+  <div id="tabPanels">
+    <div id="t1" role="tabpanel">
+    </div>
+    <div id="t2" role="tabpanel">
+    </div>
+  </div>
+`;
+
+describe('TabInterface', () => {
+  let tabList;
+  let tab1;
+  let tab2;
+  let panel1;
+  let panel2;
+  let tabInterface;
+
+  beforeEach(() => {
+    document.body.innerHTML = BASIC_TAB_HTML;
+    tabList = document.getElementById('tabs');
+    tab1 = document.getElementById('t1-label');
+    tab2 = document.getElementById('t2-label');
+    panel1 = document.getElementById('t1');
+    panel2 = document.getElementById('t2');
+    tabInterface = new TabInterface(tabList);
+  });
+
+  afterEach(() => {
+    document.getElementsByTagName('html')[0].innerHTML = '';
+    tabInterface = null;
```

Comment:
Since this is also reset in the `beforeEach`, I don’t think the cleanup here is necessary.